package.json

```diff
@@ -1,61 +1,83 @@
 {
-  "name": "@metamask/module-template",
-  "version": "0.0.0",
-  "description": "The MetaMask Node module template.",
+  "name": "@metamask/openrpc-inspector-transport",
+  "version": "0.0.0-development",
+  "homepage": "https://metamask.github.io/iframe-ee-openrpc-inspector-transport",
+  "description": "",
+  "main": "build/index.js",
   "repository": {
     "type": "git",
-    "url": "https://github.com/MetaMask/metamask-module-template.git"
+    "url": "https://github.com/MetaMask/iframe-ee-openrpc-inspector-transport.git"
   },
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
-  "files": [
-    "dist/"
-  ],
   "scripts": {
     "setup": "yarn install && yarn allow-scripts",
-    "test": "jest",
-    "test:watch": "jest --watch",
-    "prepublishOnly": "yarn build:clean && yarn lint && yarn test",
+    "start": "yarn typings && webpack serve --progress --mode='development'",
+    "build": "yarn typings && webpack --mode production",
+    "typings": "open-rpc-typings -d src/openrpc.json --output-ts src/__GENERATED_TYPES__/",
     "lint:eslint": "eslint . --cache --ext js,ts",
     "lint:misc": "prettier '**/*.json' '**/*.md' '!CHANGELOG.md' '**/*.yml' --ignore-path .gitignore",
     "lint": "yarn lint:eslint && yarn lint:misc --check",
     "lint:fix": "yarn lint:eslint --fix && yarn lint:misc --write",
-    "build:clean": "rimraf dist && yarn build",
-    "build": "tsc --project ."
+    "test": "yarn typings && yarn lint"
   },
+  "author": "",
```

Comment:
This field is optional, so it's probably better to omit it rather than leave it blank. I've been leaving it blank in all of the organization repos (unless it's set already).